hasjob/views/helper.py

```diff
@@ -28,14 +28,38 @@
 gif1x1 = 'R0lGODlhAQABAJAAAP8AAAAAACH5BAUQAAAALAAAAAABAAEAAAICBAEAOw=='.decode('base64')
 
 
-@app.route('/_sniffle.gif')
+@app.route('/_sniffle', methods=['POST'])
```

Comment:
This will now be `/api/1/anonsession`. This should also move into an `views/api.py`.